cypress.config.ts

```diff
@@ -1,11 +1,29 @@
 import { defineConfig } from 'cypress';
+// eslint-disable-next-line @typescript-eslint/no-require-imports -- No types exist for this package
+const webpackPreprocessor = require('@cypress/webpack-batteries-included-preprocessor');
+
+const webpackOptions = () => {
+	const options = webpackPreprocessor.getFullWebpackOptions();
+
+	return {
+		...options,
+		resolve: {
+			...options.resolve,
+			fallback: {
+				...options.resolve.fallback,
+				crypto: require.resolve('crypto-browserify'),
+			},
+		},
+	};
+};
```

Comment:
As of upgrading to Cypress v15 most Polyfills have been removed https://github.com/cypress-io/cypress/pull/31738 to avoid installing unecessary dependencies which may have vulnerabilities.

We need to re-add the `crypto` pollyfill as we need to use the `crypto` module to generate HMAC tokens in our cypress tests.